client-portal/modules/card/containers/List.tsx

```
@@ -10,83 +9,75 @@
+      ...queryConfig,
+      variables: {_id: pipelineId },
+    }
+  );
 
-  if (loadingStages || loadingLable || loadingAssignedUsers) {
+  if (loadingStages || loadingLabels || loadingAssignedUsers) {
     return <Spinner objective={true} />;
   }
 
+ 
+  const stages = stagesData && stagesData[`${type}sStages`] || [];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)